arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/Schedule.kt

```diff
@@ -458,6 +484,36 @@ public sealed class Schedule<Input, Output> {
       }
     }
 
+    override suspend fun <C> repeatAsFlow(
+      fa: suspend () -> Input,
+      orElse: suspend (Throwable) -> C
+    ): Flow<Either<C, Output>> =
+      flow {
+        val loop = Atomic(true)
```

Comment:
This should be safe as `var loop = true` since there are no parallel processes possible here.